### PR TITLE
Native YAML

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
monitoring/zabbix_maintenance

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @Fale. To the current maintainers, @eikef please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by eikef** [comment]
shipit

**Comment by eikef** [comment]
looks good to me, thank you Fale :-)

**Comment by gregdek** [comment]
Thanks again to @Fale for this PR, and thanks @eikef for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: monitoring/zabbix_maintenance.py [hunk 1]
```diff
@@ -122,40 +122,44 @@
...
-                      host_name=www1.example.com
-                      host_groups=Office,Dev
-                      state=present
-                      server_url=https://monitoring.example.com
-                      login_user=ansible
-                      login_password=pAsSwOrD
+- zabbix_maintenance:
+    name: Update of www1
+    host_name: www1.example.com
+    host_groups: Office,Dev
```
Comment (resmo) [inline monitoring/zabbix_maintenance.py]: this should be a yaml list

File: monitoring/zabbix_maintenance.py [hunk 2]
```diff
@@ -122,40 +122,44 @@
...
-- zabbix_maintenance: name=update
-                      host_names=www1.example.com,db1.example.com
-                      state=present
-                      collect_data=false
-                      server_url=https://monitoring.example.com
-                      login_user=ansible
-                      login_password=pAsSwOrD
+- zabbix_maintenance:
+    name: update
+    host_names: www1.example.com,db1.example.com
```
Comment (resmo) [inline monitoring/zabbix_maintenance.py]: this should be a yaml list

**Comment by Fale** [comment]
Thanks, fixed :)

**Pull request merged**